src/config.js

```diff
@@ -203,6 +203,10 @@ export default class Config {
       cert: String(opts.cert || this.getOption('cert') || ''),
       key: String(opts.key || this.getOption('key') || ''),
     });
+
+    if (this.getOption('production') || process.env.NODE_ENV === 'production') {
```

Comment:
We had a bug recently in React Native.
One of the scripts that calls `yarn install` had NODE_ENV = production and I could not override this with `--no-production`.
Should we open a separate issue for that?